Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -20,15 +19,20 @@ public class CameraManager {
     }
 
     /// Pointer HashTable for holding camera animators
-    public internal(set) var cameraAnimators = NSHashTable<CameraAnimator>.weakObjects()
+    internal var cameraAnimators = NSHashTable<CameraAnimator>.weakObjects()
+
+    /// List of animators currently alive
+    public var cameraAnimatorsList: [CameraAnimator] {
```

Comment:
Are there any memory semantics to be concerned about here? Could a developer inadvertently keep the camera animators alive?

Reply:
I don't see this as being meaningfully different from what we had before. Certainly, yes, a developer who stores the array would keep the animators alive. They could trivially do the same thing with the old API.

Reply:
Hmm, I agree with you that it shouldn't be a problem. CameraAnimators being kept alive by the developer isn't really a bad thing.